src/main/java/liuni/TwitterConfig.java

```diff
@@ -0,0 +1,91 @@
+package liuni;
+
+import org.hibernate.validator.constraints.NotEmpty;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+import javax.validation.constraints.NotNull;
+
+public class TwitterConfig extends io.dropwizard.Configuration {
+    @NotNull
+    private Map<String, String> twitter;
```

Comment:
Don't use a map as then you've got all of the string constants below

Reply:
I'm not sure I understand? The constants below are the key names, not the values.